.github/workflows/ci.yml

```diff
@@ -5,40 +5,35 @@ on:
     branches:
       - "**"
 
-env:
-  PNPM_CACHE_FOLDER: .pnpm-store
-
 jobs:
   build:
     name: CI
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
       - name: Setup Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version: 16
 
       # pnpm stuff
-      - name: pnpm — install
-        run: npm i pnpm@latest -g
-      - name: pnpm - config
-        run: pnpm config set store-dir $PNPM_CACHE_FOLDER
-      - name: pnpm - install
-        run: pnpm install
+      - name: Log NPM version
+        run: npm -v
```

Comment:
up to you, but IMO this is not neccessary

Reply:
```suggestion
```